wallet/wallet_interface.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package wallet
+
+import (
+	"context"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/account"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/tx"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/network"
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+type BuildTxParams struct {
+	BuildTxInput
+	Account account.Account
```

Comment:
Lets make `Account` a pointer so we don't create many copies of the struct (same in other `Params` structs)

Reply:
Placed this comment on https://github.com/ava-labs/avalanche-tooling-sdk-go/pull/172, and will be addressed there